src/renderer/component/channelTile/view.jsx

```diff
@@ -13,6 +13,7 @@ type Props = {
   claim: ?{
     claim_id: string,
     name: string,
+    permanent_url: string,
```

Comment:
This should use the `Claim` type in `types/claim`

Reply:
Sorry, didn't notice that @seanyesmunt  :)

Reply:
Changed 🤘